templates/docs/examples/templates/typographic-spacing.html

```diff
@@ -1223,43 +1237,57 @@ <h6 class="p-card__title">Title</h6>
     <div class="col-2">
       <h1>H1</h1>
       <div class="p-card">
-        <h1 class="p-card__title">Title</h1>
+        <div class="p-card__inner">
```

Comment:
If that's commented out I'd suggest dropping it completely. Seems like we don't need these examples. And if we need them we should uncomment them.